lib/src/value.dart

```diff
@@ -177,6 +179,13 @@ abstract class Value {
   SassFunction assertFunction([String? name]) =>
       throw SassScriptException("$this is not a function reference.", name);
 
+  /// Throws a [SassScriptException] if [this] isn't a mixin reference.
+  ///
+  /// If this came from a mixin argument, [name] is the argument name
```

Comment:
This should still be "function argument", since the Dart API doesn't support defining custom mixins.